PICMI_Python/lasers.py

```diff
@@ -65,6 +66,65 @@ def __init__(self, wavelength, waist, duration,
         self.handle_init(kw)
 
 
+class PICMI_AnalyticLaser(_ClassWithInit):
+    """
+    Specifies a laser with an analytically described distribution
+      - field_expression: Analytic expression describing the electric field of the laser(string) [V/m]
+                            Expression should be in terms of the position, 'X', 'Y', in the plane orthogonal
+                            to the propagation direction, and 't' the time. The expression should describe
+                            the full field, including the oscillitory component.
+                            Parameters can be used in the expression with the values given as keyword arguments.
+      - propagation_direction=[0,0,1]: Direction of propagation (unit vector) [1]
+      - polarization_direction=[1,0,0]: Direction of polarization (unit vector) [1]
+
+      Even though the parameters below should be built into the expression, some codes require
+      specified values for numerical purposes.
+
+      - wavelength: Laser wavelength
+      - amax: Maximum normalized vector potential
+              Specify either amax or Emax (Emax takes precedence).
+      - Emax: Maximum amplitude of the laser field [V/m]
+              Specify either amax or Emax (Emax takes precedence).
```

Comment:
Hi @dpgrote ,
Thanks for this super nice PR!
I only have super minor suggestions:

```suggestion
      - amax: Maximum normalized vector potential
      - Emax: Maximum amplitude of the laser field [V/m]
      Specify either amax or Emax (Emax takes precedence).
```

Reply:
Thanks for the comments! Yes, this is a nice simplification. I'll commit it.